SoundBuffered.js

```diff
@@ -301,6 +307,8 @@ SoundBuffered.prototype._play = function (pitch) {
 		return;
 	}
 
+	if (this.source) this.source.disconnect(this.sourceConnector);
```

Comment:
The only thing I could say is that you explicitly set `onended` to `null` in a case above, but not here. I'm sure the GC will do its job, but at the same time, memory leakage is what these recent PRs are all about :)